packages/shopify-codemod/test/fixtures/mocha-context-to-closure/contextually-declared-functions.input.js

```diff
@@ -0,0 +1,12 @@
+suite('top-level', function() {
+  setup('name', () => {
+    this.foo = 'foo';
+    this.bar = function() {
+      return this.foo === 'foo';
```

Comment:
If it's a function assigned to the mocha context, it's `this` will be the mocha context, so it's safe to covert those.